docs/SDK/Explanation/authentication/sessionSigs.md

```diff
@@ -136,23 +249,50 @@ const decryptedString = await decryptedFiles["string.txt"].async(
 console.log("decrypted string", decryptedString);
 ```
 
-## Obtaining session signatures in NodeJS
+## Obtaining the SessionSig in NodeJS
 
 You can use any wallet or signing method with session signatures because the `getSessionSigs()` function supports passing a callback called `authNeededCallback` that will be fired when a wallet signature is needed. You can see an example of this here: https://github.com/LIT-Protocol/js-serverless-function-test/blob/main/js-sdkTests/sessionKeys.js#L31
 
 The `getSessionSigs()` function will generate a session key for you automatically and attempt to store it in LocalStorage. In case you have not polyfilled LocalStorage, you may instead generate the session key yourself using `generateSessionKeyPair()` and store it however you like. You can then pass it to `getSessionSigs()` as the `sessionKey` param.
 
-## Obtaining session signatures when the user doesn't have a wallet
+## Obtaining the SessionSig when user doesn't have a wallet
 
-You can use Oauth login with services including Discord and Google when the user doesn't have a wallet. You can see an example of how to do this using Google Oauth here: https://github.com/LIT-Protocol/oauth-pkp-signup-example/blob/main/src/App.tsx#L386
+You can use Oauth login with services including Discord and Google when the user doesn't have a wallet. You can see an example of how to do this using Google Oauth here: https://github.com/LIT-Protocol/oauth-pkp-signup-example/blob/main/src/App.tsx#L50
```

Comment:
let's update this to have the link with [here](), vs having the full link in the doc